Sources/PointFree/StripeHook.swift

```diff
@@ -10,6 +10,10 @@ import Styleguide
 
 let stripeInvoiceWebhookMiddleware =
   validateStripeSignature
+    <<< filterMap(
+      ^\Stripe.Event<Stripe.Invoice>.data.object.subscription >>> pure,
```

Comment:
Turns out that the root `subscription` on our invoices is optional! Even though we only deal with subscriptions, invoices that deal with proration push this info down into the line items 🤷‍♂️